test/e2e/debug-server.spec.ts

```
@@ -0,0 +1,49 @@
+test.describe('Server Debugging', () => {
+  test('should verify server is responding', async ({ page, request }) => {
+    // Test if server root is accessible
+    const rootResponse = await request.get('/')
+    console.log(`Root response status: ${rootResponse.status()}`)
+    console.log(`Root response headers:`, await rootResponse.headersArray())
+
+    expect(rootResponse.ok() || rootResponse.status() === 304).toBeTruthy()
+
+    // Test if error-test route exists
+    const errorTestResponse = await request.get('/error-test')
+    console.log(`/error-test response status: ${errorTestResponse.status()}`)
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)